### DISCUSSION TITLE
is it possible to make qutebrowser's window black in gnome?

### DISCUSSION THREAD

**Discussion by iambumblehead** [qutebrowser/qutebrowser#6461] [Q&A] [answered]
Starting qutebrowser this way does not affect any theme change and it opens with adwaita-dark appearance. Any suggestions?
```bash
$ GTK_THEME=Darcyan qutebrowser
```

**Comment by toofar** [comment]
QT_STYLE_OVERRIDE might be a better bet since it is a Qt application. But that'll only affect native dialogs and such. qutebrowser has its own settings for colors of things.

**Reply by iambumblehead** [reply]
I tried QT_STYLE_OVERRIDE but could not get the window title and border area to change. The window-decoration is gtk-looking and does look the way the rest of the gnome desktop is themed, Adwaita dark (good). If I change the gnome desktop theme, qutebrowser appears with the theme instantly (good), however want a custom dark theme applied to qutebrowser only and am unable to do it.

**Reply by rien333** [reply]
Have you tried launching with `QT_QPA_PLATFORMTHEME` yet? I have it set to `gtk3`.

**Reply by iambumblehead** [reply]
@rien333 I tried these now, but qutebrowser opens with the same 'adwaita' theme used by the other applications
```bash
QT_QPA_PLATFORMTHEME=gtk3 GTK_THEME=Darcyan qutebrowser
QT_QPA_PLATFORMTHEME=Darcyan GTK_THEME=Darcyan qutebrowser
```

**Reply by iambumblehead** [reply]
```bash
QT_QPA_PLATFORM=wayland qutebrowser // QT
QT_QPA_PLATFORM=wayland QT_QPA_PLATFORMTHEME=gnome qutebrowser // GTK3
```
started this way, qutebrowser uses Qt style window decorations, but they are bright white. If the decorations were black that would resolve the issue for me.

**Comment by iambumblehead** [comment, marked as answer]
this solution works for me now. Install qt5ct and create a custom dark theme with that.
```bash
QT_QPA_PLATFORM=wayland QT_QPA_PLATFORMTHEME=qt5ct qutebrowser
```

a screenshot of gtk3/adwaita-dark beside dark window qt5ct
![qutebrowser-gtk3-qt5ct](https://user-images.githubusercontent.com/2058705/122869462-2da20700-d2e1-11eb-9d42-c460e91a63f0.png)
